### DISCUSSION TITLE
SCSS support?

### DISCUSSION THREAD

**Discussion by ggmoritz** [DioxusLabs/dioxus#155] [General]
How hard would it be to make an application with SCSS? Is that already posible or maybe planned?

**Comment by jkelleyrtp** [comment]
> How hard would it be to make an application with SCSS? Is that already posible or maybe planned?

I believe trunk supports scss https://trunkrs.dev/assets/ but I've never used it so I can't say one way or the other. I think we would need to add some additional parsers of the raw rust code to support that.

**Comment by ChristopherBiscardi** [comment]
With a build.rs, I'm including tailwind output in a style tag at the moment. I assume the same would work for scss.

```rust
use dioxus::prelude::*;

static STYLES: &'static str =
    include_str!("../dist/output.css");

fn main() {
    dioxus::desktop::launch(app);
}

fn app(cx: Scope) -> Element {
    cx.render(rsx! (
        div {
            style {
                vec![STYLES]
            }
            div {  "class": "text-3xl font-bold underline","Hello, world!" }
        }
    ))
}
```

```rust
use duct::cmd;

fn main() {
    cmd!(
        "npx",
        "tailwindcss",
        "-i",
        "./input.css",
        "--content",
        "./src/**/*.rs",
        "-o",
        "./dist/output.css"
    )
    .read()
    .unwrap();
}
```

**Comment by jkelleyrtp** [comment]
Thanks @ChristopherBiscardi I'm going to add your code into our example/templates repository!

Since there's nothing I can quite do with this issue, I'll move it into the discussions pages.

**Comment by mrxiaozhuox** [comment]
If the `SCSS` can compile to the `css`, maybe will can add some `custom-command` and `auto-compile` tool in the cli. @jkelleyrtp

**Reply by mrxiaozhuox** [reply]
after the https://github.com/DioxusLabs/cli/pull/7 Review, I will add the custom-command tool.
